src/pages/workspace/accounting/netsuite/advanced/NetSuiteCustomFormIDPage.tsx

```diff
@@ -0,0 +1,101 @@
+import {useRoute} from '@react-navigation/native';
+import React, {useCallback} from 'react';
+import {View} from 'react-native';
+import ConnectionLayout from '@components/ConnectionLayout';
+import FormProvider from '@components/Form/FormProvider';
+import InputWrapper from '@components/Form/InputWrapper';
+import type {FormInputErrors, FormOnyxValues} from '@components/Form/types';
+import TextInput from '@components/TextInput';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as Connections from '@libs/actions/connections/NetSuiteCommands';
+import * as ErrorUtils from '@libs/ErrorUtils';
+import Navigation from '@libs/Navigation/Navigation';
+import * as ValidationUtils from '@libs/ValidationUtils';
+import type {ExpenseRouteParams} from '@pages/workspace/accounting/netsuite/types';
+import type {WithPolicyConnectionsProps} from '@pages/workspace/withPolicyConnections';
+import withPolicyConnections from '@pages/workspace/withPolicyConnections';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+
+function NetSuiteCustomFormIDPage({policy}: WithPolicyConnectionsProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const policyID = policy?.id ?? '-1';
+    const route = useRoute();
+    const params = route.params as ExpenseRouteParams;
+    const isReimbursable = params.expenseType === CONST.NETSUITE_EXPENSE_TYPE.REIMBURSABLE;
+
+    const config = policy?.connections?.netsuite?.options.config;
+
+    const exportDestination =
+        (isReimbursable ? config?.reimbursableExpensesExportDestination : config?.nonreimbursableExpensesExportDestination) ?? CONST.NETSUITE_EXPORT_DESTINATION.EXPENSE_REPORT;
+    const customFormIDKey = isReimbursable ? 'reimbursable' : 'nonReimbursable';
```

Comment:
Can we make consts for these so that someone doesn't one day mistakenly change `nonReimbursable` to `nonreimbursable`?

(We should actually change to this to be the non-camel case for so it's unified but now's not the time I'd say)